universal-application-tool-0.0.1/app/services/program/ProgramService.java

```diff
@@ -0,0 +1,17 @@
+package services.program;
+
+import com.google.common.collect.ImmutableList;
+
+/** Operations you can perform on Programs. */
+public interface ProgramService {
+
+    /** List all programs. */
+    public ImmutableList<ProgramDefinition> listProgramDefinitions();
```

Comment:
Suggestion: 💁🏻 All abstract, default, and static methods in an interface are implicitly public, so you can omit the public modifier.

Reply:
Done!